lms/djangoapps/grades/tasks.py

```diff
@@ -58,12 +61,24 @@ class _BaseTask(PersistOnFailureTask, LoggedTask):  # pylint: disable=abstract-m
     abstract = True
 
 
-@task
-def compute_grades_for_course(course_key, offset, batch_size):  # pylint: disable=unused-argument
+@task(base=_BaseTask)
+def compute_grades_for_course(course_key, offset, batch_size):
```

Comment:
Can you update this function signature to take `kwargs` instead?  Passing `kwargs` should now be the best practice for all tasks we create in the future.  Otherwise, we always run into backward compatibility issues as @efischer19 and I have encountered.  (I was thinking of adding a parameter to this task, related to my PR, but ran into this issue.)
Also, I believe the task isn't retried unless it's encased in a `try-except` block with a call to `self.retry`.  Please consult @efischer19 on this tomorrow.